packages/backend/src/tests/walletAddress.ts

```diff
@@ -31,7 +32,10 @@ export async function createWalletAddress(
   const walletAddressService = await deps.use('walletAddressService')
   const walletAddressOrError = (await walletAddressService.create({
     ...options,
-    assetId: options.assetId || (await createAsset(deps)).id,
+    assetId:
+      options.assetId ||
+      (await createAsset(deps, undefined, options.tenantId)).id,
+    tenantId: options.tenantId || (await createTenant(deps)).id,
```

Comment:
If I'm not mistaken, if we don't pass in `options.tenantId`, we will end up creating an asset and a wallet address with different `tenantId`s

Reply:
Correct.

Reply:
I don't think that could/should happen realistically. Wallet address tenantId should match tenantId for the asset.